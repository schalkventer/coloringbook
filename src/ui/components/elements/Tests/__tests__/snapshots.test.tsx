import React from 'react';
import { create } from 'react-test-renderer';
import { Default } from './stories';
import 'jest-styled-components';

// ⚙️ Elements
// 🔨 Custom
// 📺 Views

describe('UI Components', (): void => {
  describe('Elements', (): void => {
    describe('Tests', (): void => {
      describe('snapshots', (): void => {
        test('Default', (): void => {
          expect(create(<Default />).toJSON()).toMatchSnapshot();
        });
      });
    });
  });
});
