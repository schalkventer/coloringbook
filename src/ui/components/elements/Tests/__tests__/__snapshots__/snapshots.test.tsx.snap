// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI Components Elements Tests snapshots Default 1`] = `
.c0 {
  font-size: 14px;
  font-family: "Nunito Sans",-apple-system,".SFNSText-Regular","San Francisco",BlinkMacSystemFont,"Segoe UI","Helvetica Neue",Helvetica,Arial,sans-serif;
}

<ul>
  <li
    className="c0"
  >
    ...
     
     
    
  </li>
</ul>
`;
